### PR TITLE
Bugfix/adding css handdles

### PR DESCRIPTION
**Pull request by BeatrizMiranda**
#### What problem is this solving?

Add new css handles to customize filter navigator on mobile 

#### How to test it?

[handdle](https://handdle--equishopper.myvtex.com/riding-apparel)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -326,6 +326,8 @@ In order to apply CSS customization in this and other blocks, follow the instruc
 | CSS handles                           |
 | ------------------------------------- |
 | `accordionFilter`                     |
+| `filterBreadcrumbsContent`            |
+| `filterBreadcrumbsText`               |
```
Comment (guerreirobeatriz) [inline docs/README.md:330]:
```suggestion
```

File: react/components/AccordionFilterItem.js [hunk 2]
```diff
@@ -54,10 +56,13 @@ const AccordionFilterItem = ({
             onClick={onOpen}
           >
             <div
-              className={classNames('pv4 c-on-base', {
-                't-small': open,
-                't-heading-5': !open,
-              })}
+              className={classNames(
+                `${handles.accordionFilterContent} pv4 c-on-base`,
```
Comment (Klynger) [inline react/components/AccordionFilterItem.js:60]:
```suggestion
                handles.accordionFilterContent, 'pv4 c-on-base',
```

Comment (Klynger) [reply react/components/AccordionFilterItem.js:60]: This one is missing too

File: react/components/AccordionFilterItem.js [hunk 3]
```diff
@@ -37,7 +39,7 @@ const AccordionFilterItem = ({
   return (
     <Fragment>
       {!open && (
-        <div className="pl7">
+        <div className={`${handles.accordionFilterContainer} pl7`}>
```
Comment (Klynger) [inline react/components/AccordionFilterItem.js:42]: I think this handle is missing in the documentation

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
```
Comment (klzns) [inline CHANGELOG.md:9]:
```suggestion
### Added
```

**Comment by klzns** [comment]
Thanks for your contribution! We intend to merge this tomorrow.

**Pull request merged**